data/hai/hai missions.txt

```diff
@@ -1932,3 +1932,73 @@ mission "Station Exploration Abandon Ship"
 			label fine
 			`	"You are being fined <fine> for abandoning your passengers without warning. You're lucky you didn't end up with a more serious punishment." After further chewing you out for endangering your passengers' safety, he walks away from you in a huff.`
 
+mission "Hai Pilgrimage"
+	name "Hai spiritual leader to <planet>"
+	description "Bring a Hai spiritual leader to <destination>. He will pay you <payment>."
+	minor
+	source
+		government "Hai"
+		not attributes "station"
+	destination
+		attributes "unfettered"
+	passengers 1
+	to offer
+		random < 30
+		has "outfit: Quantum Keystone"
```

Comment:
Should this offer if you have the remnant key stone? Does the Hai know the difference?

Reply:
This might be a good place to address the difference directly again with some branching and a defer, but maybe in another PR to hash out the specifics?

The campaign diplomat also recognizes the difference, iirc?

Reply:
the description of the hai one explains the difference nowadays, its displayed in evidence or smth

Reply:
I have the feeling that Hareek would be able to tell the difference. I'll try adding a branch for if the player has a Remnant keystone installed instead.

Reply:
Added a branch for if the player has a Remnant keystone.